packages/http/src/Session/Session.php

```diff
@@ -41,6 +41,7 @@ final class Session
     public function __construct(
         public SessionId $id,
         public DateTimeInterface $createdAt,
+        public DateTimeInterface $lastActiveAt,
```

Comment:
Will this change be backwards compatible? It's ok if the current session becomes invalid, but there shouldn't be any errors